tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -587,12 +587,22 @@ def _write_and_random_read(self, scale: ScaleParameters, topic_names):
         self.logger.info(
             "Write+randread stress test complete, verifying sequentially")
 
+        max_msgs = None
+        expect_transmit_time = 600  # empirically derived
```

Comment:
Ah my comment got swallowed, though this still LGTM.

I would've expected this to be set to `None` if trying to revert the behavior to before https://github.com/redpanda-data/redpanda/pull/15287/files#diff-1918605fe4f7ee419c1af283695cfb29917fbad61e10faafa40623c051a5e0c4L590

That said, looking around, I'm not sure where `None` is handled gracefully, and 600 seems much better

Reply:
You are right. I'd leave it as-is as explicit value here is a tad more readable.